docs/getting-started/quick-start.md

```diff
@@ -144,6 +148,9 @@ Lastly, add a tag for the editor to attach to.
 
 Your final page should look similar to the one below.
 
+<details>
+<summary>Final build</summary>
```

Comment:
I don't like `build` world :grin:

Reply:
"Final solution" is both much better and much worse.